Script/AtomicNET/AtomicNET/Core/Profiler.cs

```diff
@@ -12,12 +13,16 @@ public partial class Profiler : AObject
         {
 #if ATOMIC_PROFILING
             var profiler = AtomicNET.GetSubsystem<Profiler>();
-            profiler?.BeginBlock(name, file, line, color, (byte)status);
+            if (profiler != null)
```

Comment:
I mean so don't need to call GetSubsystem per block, which is not a terribly expensive call, though still, if that gets ugly that's fine, can cross bridge if it is an issue 🐫